### ISSUE TITLE
Incorrect data when array has a default value

### ISSUE THREAD

**Issue by unclejustin** [luisrudge/prisma-generator-fake-data#15]
The problem with this schema is not supported by sqlite so the sandbox does not work.

Sample schema with problem:

```
model User {
  id            String            @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
  roles         UserRole[]        @default([USER])
}

enum UserRole {
  ADMIN
  USER
 }
```

This produces the following fake data:

```
export function fakeUserComplete() {
  return {
    id: faker.string.uuid(),
    roles: USER,
  };
}
```

It should produce:

```
export function fakeUserComplete() {
  return {
    id: faker.string.uuid(),
   roles: faker.helpers.arrayElement([
      UserRole.USER,
    ] as const),
  };
}
```

This actually breaks our build 😢 

Using "prisma-generator-fake-data": "^0.11.1" with postgresql provider.

**Comment by luisrudge** [comment]
Interesting. What version were you using before? 
This is the only change from 0.11 to 0.11.1:
<img width="455" alt="image" src="https://github.com/luisrudge/prisma-generator-fake-data/assets/941075/86785e40-15f1-453d-a487-de07efdc59fc">

**Comment by unclejustin** [comment]
Just submitted a PR that fixes it for me locally.

**Comment by unclejustin** [comment]
We were on version 0.10.0 before.

**Comment by luisrudge** [comment]
So weird.. I didn't touch the enum part between 0.10 and 0.11 https://github.com/luisrudge/prisma-generator-fake-data/compare/v0.10.0...v0.11.1 🤔

**Comment by unclejustin** [comment]
Maybe no one has hit this edge case before 🤷

**Comment by luisrudge** [comment]
Yeah, I just noticed it was an enum array. Not sure we handled this at all before 😅  maybe it worked by accident haha. I'll get your PR merged and released today 🎉  Thanks for the help

**Comment by luisrudge** [comment]
Released in 0.12.0. Thanks for the help!

**Comment by unclejustin** [comment]
Thanks for the quick turnaround! 🙌
